tox.ini

```diff
@@ -32,7 +32,7 @@ commands = flake8
 ignore = W504
 show-source = True
 max-line-length = 140
-exclude = ./.*,build,dist,*egg,venv, git_machete/docs.py
+exclude = ./.*,build,dist,*egg,venv, git_machete/handmade_short_docs.py, git_machete/generated_docs.py
```

Comment:
`handmade_short_docs` excluded due to line length right?

Reply:
@PawelLipski Yes